package.json

```diff
@@ -1,25 +1,26 @@
 {
   "private": true,
   "scripts": {
-    "eslint": "eslint --max-warnings 0 --ext .js ."
+    "eslint": "eslint --max-warnings 0 --ext .js .",
+    "webpack": "webpack --watch --config config/webpack.js"
```

Comment:
I was complaining because it's not really needed, and I'm sure that this alphabetical order won't be preserved over time 😀 Anyways, I don't want to be a blocker. Merging...